Use v2 by default for historic_agg_v2

shlomikushchi opened the pull request:
* use the V2 url path from https://polygon.io/docs
* use dateutil to format the dates properly
* documented historic_agg_v2()
* removed historic_agg() (v1) to avoid confusion.
* removed tests for historic_agg()

umitanuki commented:
That would break other methods. Can you rather remove v1 historic agg to avoid confusion?

shlomikushchi commented:
* also renoved historic_agg() (v1) to avoid confusion.
* removed tests for historic_agg()
* changed default to v1 in the rest.get()  method
* updated version in init file (0.47)

shlomikushchi commented on alpaca_trade_api/polygon/rest.py:
```diff
@@ -109,27 +110,28 @@ def historic_quotes_v2(
+        :param limit: max samples to retrieve (seems like we get "limit - 1" )
+        :return:
+        """
+        path = '/aggs/ticker/{symbol}/range/{multiplier}/{timespan}/{_from}/{to}'.format(symbol=symbol,
```
please note that some of our code that calls this method (e.g pylivetrader)
is calling it like this:
```py
self._api.polygon.historic_agg_v2(
                symbol, 1, size,
                int(_from.timestamp()) * 1000,       <====================
                int(to.timestamp()) * 1000      <====================
            )
```
and it doesn't work because this code expects a string.

so we should either fix all calling code to use the same format, or make this code adjustable to this scenario.

umitanuki commented:
@shlomikushchi so this is backward compatible?
